src/main/java/picard/util/IntervalListTools.java

```diff
@@ -448,22 +449,19 @@ protected int doWork() {
             }
         }
 
-        final IntervalList output = new IntervalList(header);
-        finalIntervals.forEach(output::add);
+        final IntervalList outputIntervals = new IntervalList(header);
+        finalIntervals.forEach(outputIntervals::add);
 
         final ScatterSummary resultIntervals;
 
         if (SCATTER_CONTENT != null) {
-            final long listSize = SUBDIVISION_MODE.make().listWeight(output);
+            final long listSize = SUBDIVISION_MODE.make().listWeight(outputIntervals);
             SCATTER_COUNT = (int)Math.round((double) listSize / SCATTER_CONTENT);
             LOG.info(String.format("Using SCATTER_CONTENT = %d and an interval of size %d, attempting to scatter into %s intervals.", SCATTER_CONTENT, listSize, SCATTER_COUNT));
         }
 
         if (OUTPUT != null && SCATTER_COUNT > 1) {
-
-            IOUtil.assertDirectoryIsWritable(OUTPUT);
```

Comment:
@cmnbroad @droazen I'm removing these directory writable checks because it always throws an error in google cloud (directories 'don't exist' in gcloud…) Let me know if you think it's problematic